.ci/install_runtime.sh

```diff
@@ -41,6 +41,9 @@ export PKGLIBEXECDIR=/usr/libexec/clear-containers
 # For the pause bundle
 export LOCALSTATEDIR=/var
 
+# Qemu Binary with Q35 support
+export QEMUCMD=qemu-cc-system-x86_64
```

Comment:
this will be used when building the `cc-runtime`
https://github.com/clearcontainers/runtime/blob/master/Makefile#L83